### PR TITLE
Somashekars/wip 1

### PR DESCRIPTION
**Pull request by sandesh-vmw**
Modifying VM actions and two thin

### CODE REVIEW

**Comment by sandesh-vmw** [comment]
vm actions

File: vm-service/README.md [hunk 1]
```diff
@@ -136,6 +136,23 @@ Click on the 3 vertical dots to open the menu.
 
 ![VM Grid Menu](source/images/vm-service-grid-menu.png "VM Grid Menu")
 
+#### Publishing the VM to a writable Content Library
+![VM Publish Action](source/images/vm-service-publish-action.png "Starting VM Publish") ![VM Publish Action](source/images/vm-service-publish-modal.png "Starting VM Publish")
+For namespaces backed by vCenter 8.0 Update2 and above, VirtualMachines can be published as VM Templates into an namespace associated content library resource. The content library object on the vCenter must be made writable by the VI admin managing the namespace on the vCenter.
```
Comment (rrick-vmw) [inline vm-service/README.md]: Update2 => Update 2

File: vm-service/README.md [hunk 2]
```diff
@@ -136,6 +136,23 @@ Click on the 3 vertical dots to open the menu.
...
+
+#### Opening the web console for the VM
+![Open Web Console Action](source/images/vm-service-webconsole-action.png "Open Web Console for a VM")
+Web console for a VM can be accessed by clicking the action shown in the above picture. 
+VM Web console is accessible to consumption users over the public network. The web console connection is established by proxying through the Supervisor load balancer. 
+VM's web console will be opened after a "Open VM Web Console" request has been placed in the Supervisor. After the operator has processed the request; a URL is generated and the web console of the VM opens up in a new tab (browser pop-ups must be allowed for the web console to open.)
```
Comment (rrick-vmw) [inline vm-service/README.md]: Maybe add a comment:

Note: This may take several seconds to launch.

File: vm-service/README.md [hunk 3]
```diff
@@ -136,6 +136,23 @@ Click on the 3 vertical dots to open the menu.
...
+
+![Provide supervisor cluster IP](source/images/vm-service-webconsole-ip.png "Provide supervisor cluster IP")
+Normally, consumption UI must be able to resolve the Supervisor load balancer IP address, but if it fails to do so then the above shown modal will appear where the consumption user can provide the supervisor load balancer IP address manually. VI Admin should be able to provide the necessary IP address.
```
Comment (rrick-vmw) [inline vm-service/README.md]: shown modal => modal

**Review by rrick-vmw — APPROVED** [review]

**Pull request merged**